README.md

```diff
@@ -31,11 +31,12 @@
 
 #### Compatibility
 
-| Version to install                                                                                                                      | Support                                              |
-| --------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
-| Latest (from v3) `@storyblok/svelte`                                                                                                    | Modern browsers + Node 18+                           |
-| Latest (from v3) `@storyblok/svelte` <br> + Fetch polyfill like [isomorphic-fetch](https://github.com/matthew-andrews/isomorphic-fetch) | Browsers and Node versions with no Fetch API support |
-| [Version 2](https://github.com/storyblok/@storyblok/svelte/tree/v2.5.0) `@storyblok/svelte@2`                                           | Internet Explorer support                            |
+| Version to install                                                                                                                      | Support                                                    |
+| --------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- |
+| Latest (from v4) `@storyblok/svelte`                                                                                                    | SvelteKit2, Vite 5, storyblok/js v3, Storyblok new regions |
```

Comment:
@roberto-butti do you think this would make sense here? As this table is more related to browser/compatibility support